metadata.json

```diff
@@ -10,6 +10,7 @@
   "tags": ["redis","sentinel"],
   "operatingsystem_support": [
     { "operatingsystem":"RedHat", "operatingsystemrelease":[ "6", "7" ] }
```

Comment:
Is this valid json?  I think a trailing comma is required.  Try to build the module and see if it explodes.